src/requirements/CamelJBang.ts

```diff
@@ -38,8 +38,14 @@ export class CamelJBang {
 		return new ShellExecution('jbang', [`'-Dcamel.jbang.version=${this.camelVersion}'`, 'camel@apache/camel', 'bind', '--source', source, '--sink', sink, `'${file}'`]);
 	}
 
-	public createProject(gav: string, runtime: string): ShellExecution {
-		return new ShellExecution('jbang', [`'-Dcamel.jbang.version=${this.camelVersion}'`, 'camel@apache/camel', 'export', `--runtime=${runtime}`, `--gav=${gav}`]);
+	public createProject(gav: string, runtime: string, outputPath: string): ShellExecution {
+		return new ShellExecution('jbang',
+			[`'-Dcamel.jbang.version=${this.camelVersion}'`,
+				'camel@apache/camel',
+				'export',
+				`--runtime=${runtime}`,
+				`--gav=${gav}`,
+				`'--directory=${outputPath}'`]);
```

Comment:
to double-check the behavior when there is something in the target folder and/or when using the current folder with various version of JBang because there were some issues in the past using this parameter fr another jbang command https://issues.apache.org/jira/browse/CAMEL-21082

Reply:
Tried to replicate deleting a folder content with this command and it seems to be safe. I will add the `--clean-dir=false` option to be safe.